libglusterfs/src/iobuf.c

```diff
@@ -866,8 +832,7 @@ __iobref_grow(struct iobref *iobref)
 int
 __iobref_add(struct iobref *iobref, struct iobuf *iobuf)
 {
-    int i = 0;
-    int ret = -ENOMEM;
+    int ret = -EINVAL;
```

Comment:
nit: personally, I think we can remove this entirely, change line 850 to 'return 0' and line 852 to 'return -EINVAL'

Not important, but if you have another round of this patch, I think it makes sense.